airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -303,8 +315,11 @@ def properties(self) -> Mapping[str, Any]:
         data = self._api.get(f"/properties/v2/{self.entity}/properties")
         for row in data:
             field_type = row["type"]
-            if field_type in ["enumeration", "date", "date-time"]:
-                field_type = "string"
+            if field_type not in VALID_JSON_SCHEMA_TYPES:
+                field_type = KNOWN_CONVERTIBLE_SCHEMA_TYPES.get(field_type) or None
+                if not field_type:
```

Comment:
what about `format` attribute?
I think we should have a copy of original type there. WDYT?